collections/ansible_collections/rhsiqe/skupper/roles/run_curl/tests/inventory/host_vars/west.yml

```diff
@@ -1,55 +1,18 @@
 ---
 # Kubeconfig path for west site
-kkubeconfig: "{{ ansible_env.HOME }}/.kube/config"
+kubeconfig: "{{ ansible_env.HOME }}/.kube/config"
 
 # Namespace configuration
 namespace_name: hello-world-west
 
-# Installation configuration for Skupper
-install_skupper_skupper_repository: 'https://github.com/skupperproject/skupper.git'
-install_skupper_skupper_branch: 'v2'
-install_skupper_skupper_release_name: skupper-setup
-
 # Deployment configuration for workload
-deploy_workload_workload_image: quay.io/skupper/hello-world-frontend:latest
+deploy_workload_workload_image: quay.io/rzago/lanyard:latest
 deploy_workload_deployment_name: "frontend"
 
-# Skupper site configuration
-skupper_site_include_spec: true
-skupper_site_link_access: default
-skupper_site_manifest_template: "skupper-site.yml.j2"
-
-# Installation of skupeer controller
-install_skupper_controller_skupper_controller_manifest: 'https://raw.githubusercontent.com/skupperproject/skupper/v2/cmd/controller/deploy_cluster_scope.yaml'
-
-# Consume service configuration
-consume_service_metadata_name: "backend"
-consume_service_routing_key: "backend"
-consume_service_port: 8080
-consume_service_host: "backend"
-consume_service_manifest_template: "listener.yml.j2"
-consume_service_state: present
-
-# Access grant configuration
-access_grant_name: my-grant
-access_grant_redemptions_allowed: 10
-access_grant_expiration_window: 1h
-access_grant_state: present
-access_grant_template: "access-grant.yml.j2"
-access_grant_token_template: "access_token.yml.j2"
-access_grant_output_path: ""
-
-# Expose service configuration
-expose_service_name: frontend
-expose_service_port: 8080
-expose_service_target_port: 8080
-expose_service_type: LoadBalancer
-
 # Run curl configuration
-run_curl_address: "backend:8080/api/hello"
-run_curl_image: "quay.io/rzago/curl-telnet"
+run_curl_namespace: default
+run_curl_address: "https://fakestoreapi.com/products/1"
```

Comment:
What about using some github URL, instead?  As it is running as GH actions, github.com should be available.  Also, less likely to go out of business.